src/lib/Preview.js

```diff
@@ -867,6 +888,8 @@ class Preview extends EventEmitter {
         const { apiHost, queryParams } = this.options;
         const fileVersionId = this.getFileOption(this.file.id, FILE_OPTION_FILE_VERSION_ID) || '';
 
+        Timer.start(`${LOAD_METRIC.fileInfoTime}_${this.file.id}`);
```

Comment:
Let's use `file_${this.file.id}_XXX` since we're using `file_${this.file.id}` for the internal file object key. We probably don't need to worry about file versions for now but this will disambiguate if we choose to do so in the future.

Reply:
DONE!